### PR TITLE
1341 bug unclear error with file mismatch

### PR DESCRIPTION
**Pull request by sambodeme**
## Description

This is to handle cases where users upload the wrong workbook:
- Workbook that was not generated by SF-SAC
image

- Workbook that was generated by SF-SAC but does not match the right section.
image

### CODE REVIEW

**Comment by sambodeme** [comment]
I believe the error messages table will need an additional column for the sheet name.

File: backend/audit/views.py [hunk 1]
```diff
@@ -185,9 +186,14 @@ def post(self, request, *_args, **kwargs):
 
             audit_data = self._extract_and_validate_data(form_section, excel_file)
 
+            audit_data.pop(
+                "Meta", None
+            )  # FIXME Delete this line if Meta needs to be stored in the database.
+
```
Comment (jadudm) [inline backend/audit/views.py:192]:
```suggestion
```

File: backend/schemas/source/excel/libs/Help.libsonnet [hunk 2]
```diff
@@ -67,5 +67,9 @@ local make_url = function(anchor)
     text: 'This field must be either `Y` or `N`',
     link: make_url('yorn'),
   },
-
+  // w
+  wrong_workbook_template: {
+    text: 'The workbook you are trying to upload does not match the required template',
```
Comment (jadudm) [inline backend/schemas/source/excel/libs/Help.libsonnet:72]:
```suggestion
    text: 'The workbook you uploaded is for a different section.',
```

File: backend/schemas/source/sections/AdditionalUEIs.schema.jsonnet [hunk 3]
```diff
@@ -1,7 +1,25 @@
...
 local Types = Base.Types;
 
+local Meta = Types.object {
+  additionalProperties: false,
+  properties: {
+    section_name: Types.string {
+      enum: [Sheets.section_names.ADDITIONAL_UEIS],
+    },
+    // FIXME: 2023-08-07 MSHD: The 'Version' is currently used here as a placeholder, and it is not being enforced at the moment.
+    // Once we establish a versioning pattern, we can update this and enforce it accordingly.
```
Comment (jadudm) [inline backend/schemas/source/sections/AdditionalUEIs.schema.jsonnet:13]:
```suggestion
```

**Review by jadudm — APPROVED** [review]
Nicely done.

**Pull request merged**